components/mediation/tasks/org.wso2.micro.integrator.ntask.core/src/main/java/org/wso2/micro/integrator/ntask/coordination/task/store/connector/RDMBSConnector.java

```diff
@@ -75,9 +75,8 @@ public RDMBSConnector(DataSource dataSource) throws TaskCoordinationException {
         try (Connection connection = getConnection()) {
             DatabaseMetaData metaData = connection.getMetaData();
             String databaseType = metaData.getDatabaseProductName();
-            if (!"MySQL".equals(databaseType)) {
-                throw new CoordinateTaskRunTimeException(
-                        "Not supported data base type found : " + databaseType + " . Only MySql is supported.");
+            if (!("MySQL".equals(databaseType) || "Microsoft SQL Server".equals(databaseType))) {
+                throw new CoordinateTaskRunTimeException("Not supported data base type found : " + databaseType);
```

Comment:
Does this mean SQL server is not supported for tasks? 
With this it seems only MySQL is supported.

Reply:
No, this means MySql and SQL is only supported atm. I didn't get why you refer as only MySql supported ?

Reply:
I see. This block is like `else if` block .